openapi/multi-yaml/paths/disputes@{id}@response.yaml

```diff
@@ -0,0 +1,165 @@
+            product_description:
+              type: string
+              description: a description of the product or service that was sold
+            refund_policy_disclosure:
+              type: string
+              description: documentation demonstrating that the customer was shown your refund policy prior to purchase
+            refund_refusal_explanation:
+              type: string
+              description: justification for why the customer is not entitled to a refund
+            service_date:
+              type: string
+              description: the date on which the customer received or began receiving the purchased service
+            shipping_address:
+              type: string
+              description: the address to which a physical product was shipped
+            shipping_carrier:
+              type: string
+              description: the delivery service that shipped a physical product, such as Fedex, UPS, USPS, etc. If multiple carriers were used for this purchase, please separate them with commas
+            shipping_date:
+              type: string
+              description: the date on which a physical product began its route to the shipping address
+            shipping_tracking_number:
+              type: string
+              description: the tracking number for a physical product. If multiple tracking numbers were generated for this purchase, please separate them with commas
+            duplicate_charge_original_payment_id:
+              type: string
+              description: the payment id for the prior charge which appears to be a duplicate of the disputed charge
+
+  responses:
+    "200":
+      description: Dispute response updated
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: ../components/schemas/Envelope.yaml
+              - properties:
+                  type:
+                    example: dispute response
+                  data:
+                    $ref: ../components/schemas/DisputeResponse.yaml
+
+post:
+  summary: Submit dispute response
+  description: Submits the dispute response
+  operationId: SubmitDisputeResponse
+  parameters:
+    - $ref: ../components/parameters/id-path.yaml
+    - $ref: ../components/parameters/authorization-header.yaml
+  tags:
+    - Disputes
+  requestBody:
+    required: true
+    content:
+      application/json:
+        schema:
+          required: [forfeit]
+          properties:
+            forfeit:
+              type: boolean
+              description: when true forfeits the dispute and all other parameters passed in are ignored
+            additional_statement:
+              type: string
+              description: any additional evidence or statements
+            cancelation_policy_disclosure:
+              type: string
+              description: an explanation of how and when the customer was shown your refund policy prior to purchase
```

Comment:
Same here: Should this description refer to cancelation instead of refund?

Reply:
Same as the other ones